packages/lesmis-server/src/routes/ReportRoute.ts

```diff
@@ -20,15 +21,30 @@ export class ReportRoute extends Route {
   }
 
   async buildResponse() {
-    const { reportCode } = this.req.params;
-    const { type } = this.req.query;
+    const { entityCode, reportCode } = this.req.params;
+    const {type} = this.req.query;
```

Comment:
I think prettier wants the spaces

Reply:
🤦‍♂️🤦‍♂️ I did this as an obvious mistake to check if prettier was running after I adjusted my lint settings, and then failed to run it correctly from CLI